### ISSUE TITLE
Should subject-specific enhancement  of metadata be used by communities as a mechanism for achieving  FAIRness in data?

### ISSUE THREAD

**Issue by hrzepa** [FAIR-EG/consultation#15]
I would like to present the following search:

https://search.datacite.org/works?query=media:chemical\/x\-gaussian*+SubjectScheme:inchikey+subject:XZYDALXOGPZGNV-UHFFFAOYSA-M+media:chemical\/x\-mnpub*

as an example of FAIR metadata associated with a  FAIR dataset. The metadata from one of the two hits can be inspected at 

https://data.datacite.org/application/vnd.datacite.datacite+xml/10.14469/HPC/2635

and exploits the <subject> element from the  DataCite metadata schema to enhance the FAIR attributes of the dataset. 

https://search.datacite.org/works/10.14469/HPC/2635  shows the media types associated with this dataset, again part of the  FAIR attributes.

My question is whether such subject-specific FAIR enhancement  should be used by  communities as a mechanism for achieving  FAIRness?

[//]: # "==Do not write above this line==
Instructions for posting issues: 
(1) Review what is already there. Perhaps a comment to an existing issue would be more appropriate than opening a new one?
(2) Write your post below using Markdown (as per https://guides.github.com/features/mastering-markdown/ ) or just plain text.
(3) Don't worry about these introductory lines - you can leave or delete them, as they won't display anyway (you can check this via Preview).
(4) Hit the 'Submit new issue' button.
==Write below this line=="

**Comment by Daniel-Mietchen** [comment]
The above search currently gives no results for me.

**Comment by hrzepa** [comment]
Sometimes the escape characters can be corrupted.

Try

https://search.datacite.org/works?query=media:chemical\/x\-gaussian*+subjectScheme:inchikey+subject:XZYDALXOGPZGNV-UHFFFAOYSA-M+media:chemical\/x\-mnpub*

which  I have just tested and which works for me.

**Comment by Daniel-Mietchen** [comment]
Yes, confirming that it works now. Thanks!

**Comment by hrzepa** [comment]
Yes, I think the page presentation probably included page breaks, which break the search! It is a bit fragile!
